### DISCUSSION TITLE
I need say again ...

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1105] [Suggestions]
### Description

Thanks very much Double Commander creator !
Users changing from windows to Linux not have much options to use an file manager with much features.
For luck you have created Double Commander ...
Your software is amazing for Linux users.

Have an nice night for you and users too.

**Comment by ags1234** [comment]
I also am grateful for DC. Using it in Windows and Linux.
Also, waiting for fixes for the problems it still has.
